sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -1020,7 +1020,11 @@ private static async Task DownloadAndAssertAsync(Stream stream, BlobClient blob)
 
                 var buffer = new byte[count];
                 stream.Seek(i, SeekOrigin.Begin);
+#if NET6_0_OR_GREATER
```

Comment:
[CA2022: Avoid inexact read with Stream.Read](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2022) ReadAsync may return fewer bytes than requested which can result in unreliable code. ReadExactlyAsync should be used instead but it's only available for .NET6+. Since this is a test case, just leave the net462 version as-is.